canonical-web-and-design/hiring.canonical.com
# Modify api endpoint for the 1st chart (Pending applications)

## Done
Modified `api/application_review/pending` so it returns data for the pending chart
## QA
- Run server as Peter: `dotrun --env DEBUG_USER_EMAIL=peter.mahnke@canonical.com`
- View charts on http://0.0.0.0:8501/api/application_review/pending
- Verify it returns `json`
## Issue
Resolves #75
